### DISCUSSION TITLE
Please delete the spam message in one of my discussions

### DISCUSSION THREAD

**Discussion by stephanrauh** [community/community#42843] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I just received an email of a spammer participating in one of my discussions. I'd like to delete the message, but that seems to be impossible. The good news is that the message is not visible (except in my emails), but the header still reads "2 suggested answers", so the offending message still seems to be there.

Please delete the message or tell me how to do it myself.

Here's a screenshot of the notification email I've received:
<img width="1261" alt="grafik" src="https://user-images.githubusercontent.com/3045767/209949610-e949a63c-f44a-4836-b25b-c54ce8731fad.png">

Thanks in advance,
Stephan

**Comment by geckolinux** [comment]
https://github.com/community/community/discussions/42810

**Reply by stephanrauh** [reply]
Thanks for showing me the discussion. It's interesting, but it doesn't address my problem. Before I posted my question, I did some research, so I'm aware that GitHub suffers from a spam problem. I also suspect my particular spam has already been deleted. However, I want to make sure it never pops up again.

**Reply by denvitko** [reply]
I've already received two such e-mails (both ofc going directly to spam). But can't find a place where/how to report it.

**Comment by Akash1134** [comment, marked as answer]
Hi @stephanrauh @geckolinux @denvitko, GitHub is responding to concerns expressed by community members regarding recent spam posts & comments. Thanks for sharing and we appreciate your patience.
